remove_win32_bit/jep.md

```diff
@@ -1 +1,75 @@
-TODO
+# JEP NNN: Removal of Windows 32-bit Support
+
+_Author_	Bruno Borges & George Adams
+_Owner_ Bruno Borges
```

Comment:
White space issues?

Reply:
I don't think these fields are really coming from the JEP text. I think they are JIRA. I am keeping here just to remember what to put on the JIRA fields.